addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -538,93 +489,39 @@
 
     // Kanban Grouped Layout - "Create new column" column
     .o_column_quick_create {
-        flex: 0 0 $o-kanban-small-record-width + 2*$o-kanban-group-padding;
-
-        .o_quick_create_folded {
-            cursor: pointer;
-            padding: 12px 16px;
-            white-space: nowrap;
-            font-weight: $font-weight-bold;
-            @include o-hover-opacity(.7);
-            @include o-hover-text-color($text-muted, $body-color);
-        }
-
-        .o_quick_create_unfolded {
-            margin: $border-width ($o-kanban-inside-hgutter * .5) 0;
-            padding: $o-kanban-inside-vgutter $o-kanban-inside-hgutter;
-
-            @include media-breakpoint-down(md) {
-                margin: 0;
-            }
-
-            .o_kanban_header {
-                height: 50px;
-
-                .o_kanban_examples {
-                    cursor: pointer;
-                }
-
-                input, input:focus, input:hover {
-                    font-size: 16px;
-                    background: transparent;
-                }
-            }
-
-            .o_kanban_muted_record {
-                background: map-get($grays, '300');
-                height: 70px;
-                margin: 10px 0px;
-            }
-
-        }
+        flex: 0 0 calc(var(--KanbanGroup__smallColumn-width, #{$o-kanban-small-record-width}) + (2 * var(--KanbanGroup__padding-h, #{$o-kanban-group-padding})));
     }
 
     @include media-breakpoint-down(md) {
         .o_kanban_group, .o_column_quick_create {
-            flex: 1 0 90%; // don't take full width to give a hint of next/previous column
+            flex-basis: 90%; // don't take full width to give a hint of next/previous column
             scroll-snap-align: center;
             overflow-y: scroll;
 
             .o_kanban_header {
                 position: sticky; // keep it visible
-                top: 0;
-                z-index: 10; // same as the sticky ControlPanel
+                z-index: $zindex-sticky;
                 background-color: inherit;
             }
-
-            .o_kanban_header_title {
-                margin-top: 0 !important; // avoid sticky glitch when scrolling
-            }
         }
     }
 
     // Kanban UN-grouped Layout
     &.o_kanban_ungrouped {
         min-height: 100%;
-        align-content: flex-start;
-        flex-flow: row wrap;
-        justify-content: flex-start;
-
-        padding: $o-kanban-record-margin ($o-horizontal-padding - $o-kanban-record-margin);
+        padding: var(--Kanban__record-margin-v, #{$o-kanban-record-margin}) var(--Kanban__record-margin-h, #{$o-kanban-record-margin});
 
         .o_kanban_record {
-            flex: 1 1 auto;
-            width: $o-kanban-default-record-width;
-            margin: ($o-kanban-record-margin * 0.5) $o-kanban-record-margin;
+            width: var(--Kanban__record-width, #{$o-kanban-default-record-width});
```

Comment:
I think I've underestimated the complexity and the amount of logic necessary to handle variables using with this approach :/ 
It's better to define CSS variables with their SCSS counterpart directly I guess...probably it's what you already did in one of your firsts iterations, sorry for the hassle 😞
```scss
.o_kanban_renderer {
    --KanbanRecord-width: #{$o-kanban-default-record-width};
    
    .o_kanban_record {
            width: var(--Kanban__record-width);
     }
}
```